modules/custom/optional/govcms8_default_content/govcms8_default_content.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the module.
+ */
+
+use Drupal\govcms8_default_content\InstallHelper;
+use Drupal\menu_link_content\Entity\MenuLinkContent;
+
+/**
+ * Implements hook_install().
+ */
+function govcms8_default_content_install() {
+  if (!\Drupal::service('config.installer')->isSyncing()) {
+    \Drupal::classResolver()->getInstanceFromDefinition(InstallHelper::class)->importContent();
+
+    $menu_link = MenuLinkContent::create([
+      'title' => 'Styleguide',
+      'link' => [
+        'uri' => 'internal:/profiles/govcms/themes/custom/govcms8_uikit_starter/styleguide/'
```

Comment:
'uri' => 'internal:/themes/custom/govcms8_uikit_starter/styleguide/' now that the theme is a contrib module